core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.ts

```diff
@@ -38,19 +38,20 @@ export class CkEditorCopyToolbarComponent {
 
   toggleToolActive(): void {
     this.ckEditorCopyContentService.toggleCopyMode();
+    let element = this.document.querySelectorAll(
+      '.oppia-rte-editor'
+    ) as NodeListOf<HTMLElement>;
```

Comment:
use it as HTLMElement rather than as Element. focus/blur property not exist in type Element.